xDrip/Scenes/Home/Home/HomeInteractor.swift

```diff
@@ -30,6 +30,7 @@ final class HomeInteractor: HomeBusinessLogic, HomeDataStore {
     init() {
         glucoseDataWorker = HomeGlucoseDataWorker()
         glucoseDataWorker.glucoseDataHandler = { [weak self] in
+            self?.updateGlucoseCurrentInfo()
```

Comment:
When using weak `self` multiple times in a closure, use should write `guard let self = self else { return }` to avoid partial executions.